tools/c7n_gcp/c7n_gcp/resources/spanner.py

```diff
@@ -31,5 +31,40 @@ class resource_type(TypeInfo):
         @staticmethod
         def get(client, resource_info):
             return client.execute_command(
-                'get', {'name': resource_info['name']}
+                'get', {'name': resource_info['resourceName']}
+            )
+
+
+@resources.register('spanner-database-instance')
+class SpannerDatabaseInstance(ChildResourceManager):
+
```

Comment:
can you add a class doc string with a link to the rest api docs for the spanner db instance here.

Reply:
Done